scripts/update-lib-version.bash

```diff
@@ -16,7 +16,7 @@ EXAMPLES_DIRECTORY="$SCRIPT_DIRECTORY/../examples"
 pushd "$EXAMPLES_DIRECTORY" > /dev/null
 
 echo "Search for files in $(pwd)"
-# lines to update contains substring like "bpmn-visualization@0.7.0"
+# lines to update contains substring like "bpmn-visualization@0.8.0"
```

Comment:
:question: can we stop updating this value from version to version? it tends to state in the git history that something has changed in the script to manage the version bump.
Can we put `x.y.z` in this PR and never change this comment as part of version bump?